dcc-submission-ui/dictionary/app/index.html

```diff
@@ -41,9 +42,19 @@
         <script src="bower_components/regex-colorizer/regex-colorizer.js"></script>
         <script src="bower_components/d3-tip/index.js"></script>
         <script src="bower_components/jsoneditor/dist/jsoneditor.js"></script>
+        <script src="bower_components/jsondiffpatch/public/build/jsondiffpatch-full.min.js"></script>
 
-        <script src="scripts/vendor/beautify.js"></script>
+        <script>
+            var jsondiffpatch = jsondiffpatch.create({
```

Comment:
So I have to Initialize **jsondiffpatch** before I load **formatters** js files. Maybe we could move this into a different JS file and load it between the two; **jsondiffpatch-full.min.js** and  **jsondiffpatch-formatters.min.js**; files. Or if there is any other better way doing it.